components/SermonListCard.tsx

```diff
@@ -78,6 +116,44 @@ Props) => {
                 />
               )}
             <span style={{ width: '100%' }}></span>
+            <IconButton
+              style={{ color: 'lightblue' }}
+              onClick={() => setEditFormPopup(true)}
+            >
+              <EditIcon />
+            </IconButton>
+            <IconButton
+              style={{ color: 'red' }}
+              onClick={() => setDeleteConfirmationPopup(true)}
+            >
+              <DeleteIcon />
+            </IconButton>
+            <PopUp
+              title={'Are you sure you want to delete this sermon?'}
+              open={deleteConfirmationPopup}
+              setOpen={() => setDeleteConfirmationPopup(false)}
+            >
+              <div>
+                <Button
+                  variant="contained"
+                  onClick={() => {
+                    handleDelete(sermon.key).then(() =>
+                      setDeleteConfirmationPopup(false)
+                    );
+                  }}
+                  color="primary"
+                >
+                  Yes
+                </Button>
+              </div>
+            </PopUp>
```

Comment:
Add info about this action being irreversible.

If you like it, you can copy the style of a toggle and confirm like youtube
<img width="576" alt="Screen Shot 2022-09-28 at 10 48 55 PM" src="https://user-images.githubusercontent.com/21961090/192936218-28abe247-f5f4-4342-8ae7-f5b65a880613.png">